libs/langchain/pyproject.toml

```diff
@@ -204,7 +203,7 @@ langchain-community = {path = "../community", develop = true}
 [tool.poetry.extras]
 llms = ["clarifai", "cohere", "openai", "openlm", "nlpcloud", "huggingface_hub", "manifest-ml", "torch", "transformers"]
```

Comment:
I don't know 🤷‍♂️

Reply:
We want to remove a bunch of the extras in the long term since it doesn't make sense having them. short term we want to avoid breaking code that folks rely on.